src/pages/verify/integrations/javascript.mdx

```diff
@@ -0,0 +1,292 @@
+
+You will need to add this URL to the list of allowed URLs for your application.
+
+If you are developing with plain HTML and running your application locally, your callback URL might look like this:
+
+```
+http://localhost:3000/example/index.html
+```
+
+When you deploy your application to a different URL, you will also need to ensure you add that URL to the Callback URLs field in your application settings.
+
+## Install SDK
+
+You can include the library directly into your HTML file by loading it from [unpkg](https://www.npmjs.com/package/unpkg):
+
+```html
+<!-- index.html -->
+
+<script src="https://unpkg.com/@criipto/auth-js"></script>
+```
+
+Or install the package using [npm](https://npmjs.org/):
+
+```sh
+npm install --save @criipto/auth-js
+```
+
+## Initialize SDK with your application details
+
+If using the library from `unpkg`, you will have access to the global variable `CriiptoAuth`:
+
+```js
+// index.js
+
+var criiptoAuth = new CriiptoAuth({
+  domain: '{YOUR_CRIIPTO_DOMAIN}',
+  clientID: '{YOUR_CRIIPTO_APPLICATION_ID}',
+  store: sessionStorage,
+});
+```
+
+Or import the library if you are using npm:
+
+```js
+// index.js
+
+import CriiptoAuth from '@criipto/auth-js';
+
+var criiptoAuth = new CriiptoAuth({
+  domain: '{YOUR_CRIIPTO_DOMAIN}',
+  clientID: '{YOUR_CRIIPTO_APPLICATION_ID}',
+  store: sessionStorage,
+});
+```
+
+The `CriiptoAuth` constructor takes an object with the following parameters:
+
+- `domain (required, string)`: The domain your Criipto Application belongs to, such as `example.criipto.id`.
+- `clientID (required, string)`: The Client ID for your Criipto Application.
+
+Use your application's [domain and client ID](#register-your-application-in-criipto-verify) as the values for `domain` and `clientID` respectively.
+
+- `store (required)`: the library will store the JWT tokens in the specified location, `sessionStorage` or `localStorage`.
+
+[Additional authorization parameters](#authorization-parameters) can be passed in the `CriiptoAuth` constructor as well.
+
+### Authorization parameters
+
+All [authorization methods](#authorization-methods) take a set of authorization parameters. The following parameters are supported:
+
+- `redirectUri (string)`: The URL where Criipto will call back to with the result of a successful or failed authentication. It must be whitelisted in the "Callback URLs" section in your Criipto application settings.
+- `acrValues (string)`: Which eID to use for authentication, such as `urn:grn:authn:dk:nemid:poces`; a list of acceptable values can be found at `https://{YOUR_CRIIPTO_DOMAIN}/.well-known/openid-configuration`.
+- `responseType (string)`: The type of response you want to receive from Criipto Verify. The default `id_token` will return a JWT token. You can also set the value to `code` for PKCE flow.
```

Comment:
Remove this, in most cases we dont want people to configure this, just to use the defaults.